src/app/home/hero/heroimage/HeroImage.module.scss

```
@@ -0,0 +1,39 @@
+@use "../../../../common/styles/variables.module" as *;
+
+$width: 18em;
+$height: 24em;
+
+@mixin card-img {
+    width: $width;
+    height: $height;
+    background-color: $navy;
+    border-radius: 10%;
+}
+
+.hero_image {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    @include card-img;
+    z-index: 2;
+
+    &::before { 
+        display: block;
+        position: absolute;
+        content: '';
+        opacity: 100%;
+        @include card-img;
+        z-index: -1;
+        left: 8%;
+        top: -5%;
+        opacity: 30%;
+    }
+
+    &__image {
+        width: 100%;
+        height: auto;
+    }
+
+    & div { overflow: hidden; }
```

```suggestion
    div { overflow: hidden; }
```

& - is basically a parents name. In this case it's .hero-image. If we have children classes like 
.hero-image__image we have to use & because it's a part of their name. If you don't try to style classes you don't need & because nesting takes care of that. So if you simply nest div in some class - it will apply to every div that is in this class.